statistics/histogram.go

```diff
@@ -227,7 +227,8 @@ func ValueToString(value *types.Datum, idxCols int) (string, error) {
 		return value.ToString()
 	}
 	// Ignore the error and treat remaining part that cannot decode successfully as bytes.
-	decodedVals, remained, _ := codec.DecodeRange(value.GetBytes(), idxCols)
+	decodedVals, remained, err := codec.DecodeRange(value.GetBytes(), idxCols)
+	_ = err
```

Comment:
Why modify this line?

Reply:
Pass errcheck 😂

Reply:
We can add a comment here to avoid future reader's confusing